applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -79,6 +84,78 @@ static void event_handler(nrf_timer_event_t event_type, void *ctx)
 {
 }
 
+void audio_sync_timer_sync_led_on()
+{
+	nrfx_timer_compare(&timer_instance, NRF_TIMER_CC_CHANNEL3, audio_sync_timer_curr_time_get(),
+			   true);
+	nrfx_gpiote_out_task_trigger(SYNC_LED);
+}
+
+void audio_sync_timer_led_on_cmpr_time_set(uint32_t compare_value)
+{
+	nrfx_timer_compare(&timer_instance, NRF_TIMER_CC_CHANNEL2, compare_value, true);
+}
+
+void audio_sync_timer_led_off_cmpr_time_set(uint32_t compare_value)
+{
+	nrfx_timer_compare(&timer_instance, NRF_TIMER_CC_CHANNEL3, compare_value, true);
+}
+
+static int audio_sync_timer_sync_led_init(void)
+{
+	nrfx_err_t ret;
+
+	/* Connected the irq handler for timer 1 */
```

Comment:
"Connect the IRQ handler for timer 1"
I think we should implement a check (an if statement) here that checks if it actually is timer 1 that is used as audio sync timer